### PR TITLE
Resource remapping (targetting remap-bridges)

### PR DESCRIPTION
**Pull request by Devan-Kerman**

### CODE REVIEW

File: src/main/java/net/fabricmc/tinyremapper/OutputConsumerPath.java [hunk 1]
```diff
@@ -115,10 +113,7 @@ private OutputConsumerPath(Path destination, boolean isJar, boolean keepFsOpen,
 			Map<String, String> env = new HashMap<>();
 			env.put("create", "true");
 
-			FileSystem fs = FileSystems.newFileSystem(uri, env);
-			if (fs.isReadOnly()) throw new IOException("the jar file "+destination+" can't be written");
-
-			destination = fs.getPath("/");
+			destination = FileSystems.newFileSystem(uri, env).getPath("/");
```
Comment (sfPlayer1) [inline src/main/java/net/fabricmc/tinyremapper/OutputConsumerPath.java:116]: Why? This was specifically added to detect a common setup problem early. Bad rebase?

File: src/main/java/net/fabricmc/tinyremapper/OutputConsumerPath.java [hunk 2]
```diff
@@ -307,7 +205,7 @@ public void accept(String clsName, byte[] data) {
 			createParentDirs(dstFile);
 			Files.write(dstFile, data);
 		} catch (IOException e) {
-			throw new UncheckedIOException("error writing to "+dstFile, e);
+			throw new RuntimeException(e);
```
Comment (sfPlayer1) [inline src/main/java/net/fabricmc/tinyremapper/OutputConsumerPath.java:208]: UncheckedIOException was specifically added to make IOException unchecked.

**Review by sfPlayer1 — COMMENTED** [review]
Looks good except for some minor details and needing rebasing. Merge pending soon, along with merging into master.

File: src/main/java/net/fabricmc/tinyremapper/OutputConsumerPath.java (hunk 1 above)
Comment (Devan-Kerman) [reply src/main/java/net/fabricmc/tinyremapper/OutputConsumerPath.java:116]: yea I think so

**Pull request merged**